app/retail/templates/press_page.html

```diff
@@ -0,0 +1,191 @@
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html lang="en">
+  {% include 'shared/head.html' %}
+  <head>
+    <meta name="title" content="About Gitcoin: Find Open Bug Bounties & Freelance Coders">
+    <meta name="description" content="Gitcoin is a community for freelance developers to collaborate and monetize their skills
+    while working on Open Source projects through bounties paid in crypto (ETH or any ERC-20 token).">
+    <link rel="stylesheet" href="{% static "v2/css/landing_page.css" %}" />
+    <link rel="stylesheet" href="{% static "v2/css/press_page.css" %}" />
+    <script>
+    // Twitter widget is loading.
+    window.twttr = function (d, s, id) {
+    var t, js, fjs = d.getElementsByTagName(s)[0];
+    if (d.getElementById(id)) return; js = d.createElement(s); js.id = id;
+    js.src = "https://platform.twitter.com/widgets.js";
+    fjs.parentNode.insertBefore(js, fjs);
+    return window.twttr || (t = { _e: [], ready: function (f) { t._e.push(f) } });
+    }(document, 'script', 'twitter-wjs');
+
+  // When widget is ready, run masonry
+  twttr.ready(function (twttr) {
+    twttr.events.bind('loaded', function (event) {
+      $('.grid').masonry({
+        itemSelector : '.grid-item',
+        columnWidth : 300,
+        gutter: 10
+      });
+    });
+  });
+</script>
+  </head>
+  <body class="{{ active }} g-font-muli">
+    {% include 'shared/tag_manager_2.html' %}
+    <div class="container-fluid header">
+      {% include 'shared/top_nav.html' with class='d-md-flex' %}
+      {% include 'home/nav.html' %}
+      <div class="minihero row pt-5 pb-5 text-center">
+        <div class="col">
+          <h1 class="minihero__title mb-4">Press & Media Coverage</h1>
+          <div class="w-100"></div>
+        </div>
+        {% include 'shared/rain.html' with class="grey" %}
+      </div>
+    </div>
+
+    <div class="container-fluid white-light-bg">
+      <div class="row pb-1 pt-4">
+          <div class="col text-center">
+            <h4 class="">{% trans "Recent Press" %}</h4>
+          </div>
+      </div>
+      <div class="container">
+      <div class="row pb-4 pt-4">
+        <div class="col text-center">
+            <div class="d-md-flex justify-content-center">
+                <img src="https://i0.wp.com/softwareengineeringdaily.com/wp-content/uploads/2018/05/logo.png?w=321&ssl=1" alt="Software Engineering logo">
+                <div class="press_details ml-4">
+                  <span class="date">Tuesday, April 3 2018</span>
+                  <span class="title">Gitcoin: Open Source Bounties with Kevin Owocki</span>
+                  <p class="mt-2">Most technology companies rely on open source software projects. But open source software projects are often maintained by a group of people that is not affliated with any particular company. When an open source project.</p>
+                </div>
+              </div>
```

Comment:
^ indent fixes! could you double check the indentation the next time after raising PR ? It would make the review process smoother